templates/download/flavours.html

```diff
@@ -79,7 +79,7 @@ <h3 class="p-matrix__title"><a class="p-link--external" href="https://xubuntu.or
         <li class="p-matrix__item u-hide--small">&nbsp;</li>
         <li class="p-matrix__item u-hide--small">&nbsp;</li>
       </ul>
-      <p>A complete list of known flavours, editions and customizations is maintained on the <a href="https://wiki.ubuntu.com/DerivativeTeam/Derivatives">Ubuntu Wiki Derivatives Team page</a>.</p>
+      <p>A complete list of known flavours, editions and customizations is maintained on the <a class="p-link--external" href="https://wiki.ubuntu.com/UbuntuFlavors">Ubuntu Wiki's UbuntuFlavors page</a>.</p>
```

Comment:
s/customizations/customisations

Reply:
Good catch on the Americanisation. 

The comment on pointing to the flavours page... this is that page.